### DISCUSSION TITLE
Is it possible that validation functions returns complete location?

### DISCUSSION THREAD

**Discussion by dimaMachina** [graphql/graphql-js#3415] [Q&A]
Not only `start` of the `line` and `column` but also `end` of the `line` and `column`?

Related https://github.com/dotansimha/graphql-eslint/issues/825

**Comment by dotansimha** [comment]
@saihaj @IvanGoncharov what do you think?

**Reply by saihaj** [reply]
> If an error can be associated to a particular point in the requested GraphQL document, it should contain an entry with the key locations with a list of locations, where each location is a map with the keys line and column, both positive numbers starting from 1 which describe the beginning of an associated syntax element.
https://spec.graphql.org/October2021/#sel-FAPHRPDFAACHAzjS

So as per spec we only return starting position. But I can see how it is useful probably `graphiql` has something for error recovery purposes. I suppose we can file this against spec first?
